src/System.CodeDom/tests/System/CodeDom/Compiler/CodeValidatorTests.cs

```diff
@@ -0,0 +1,2713 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Xunit;
+
+namespace System.CodeDom.Compiler.Tests
+{
+    public class CodeValidatorTests
+    {
+        public static IEnumerable<object[]> ValidateIdentifiers_Valid_TestData()
+        {
+            // CodeComment.
+            yield return new object[] { new CodeComment() };
```

Comment:
if this was generated please share the code (ideally tt template), if not the fine as is

Reply:
was not generated manually